releases/release-support-policy.md

```diff
@@ -10,12 +10,13 @@ This page explains Cockroach Lab's policy for supporting [major releases](../rel
 
 Each major release of CockroachDB goes through the following support cycle:
 
-- **Maintenance Support:** For at least 365 days from the major release date, Cockroach Labs will produce regular patch releases including resolutions to problems identified by users.
+- **Maintenance Support:** For at least 365 days from the major release date, Cockroach Labs will produce regular patch releases including resolutions to problems identified by users and critical security fixes.
 
 - **Assistance Support:** Following the maintenance support period, Cockroach Labs will provide assistance support for at least an additional 180 days. During this period, the following guidelines will apply:
     - New enhancements and error corrections will not be made to the major release.
     - Cockroach Labs will direct customers to existing fixes/patches and workarounds applicable to the reported case.
     - Cockroach Labs may direct customers to [upgrade](../stable/upgrade-cockroach-version.html) to a more current version of the product if a workaround does not exist.
+    - Cockroach Labs will continue to add critical security fixes to the major release.
```

Comment:
```suggestion
    - Cockroach Labs will continue to add critical security fixes to the major release in the form of patch releases.
```

A clarification, assuming my interpretation is correct. (E.g. we wouldn't actually add the fix to v20.2.0.) I reviewed the public [CockroachDB Release Cycle](https://wiki.crdb.io/wiki/spaces/CRDB/pages/68616452/CockroachDB+Release+Cycle) page to try to confirm.